org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+  #- name: meson-perl
+  #  buildsystem: meson
+
+  #-------------------------------------------------------------------------------
+  # libpeas-perl.
+  #
+  # TODO
+  #-------------------------------------------------------------------------------
+  #- name: libpeas
+  #  buildsystem: meson
+  #  sources:
+  #    - type: git
+  #      url: 'https://gitlab.com/neva_blyad/libpeas-perl'
+  #      branch: main
+
+  #-------------------------------------------------------------------------------
+  # libgee
+  #-------------------------------------------------------------------------------
+  - name: libgee
+    buildsystem: autotools
+    build-options:
+      env:
+        - ACLOCAL_PATH=/usr/lib/sdk/vala/share/aclocal/
+    sources:
+      - type: git
+        url: 'https://gitlab.gnome.org/GNOME/libgee.git'
+        #branch: master
+        tag: '0.20.6'
+    post-install:
+      - ln -s /app/share/vala/vapi/gee-0.8.vapi /usr/lib/sdk/vala/share/vala-0.56/vapi/
+      - ln -s /app/share/gir-1.0/Gee-0.8.gir /usr/share/gir-1.0/
+    cleanup:
+      - /include
+      - /share
+      - /lib/pkgconfig
+
+  #-------------------------------------------------------------------------------
+  # wxFormBuilder
+  #-------------------------------------------------------------------------------
+  - name: wxformbuilder
+    sources:
+      - type: git
+        url: 'https://github.com/wxFormBuilder/wxFormBuilder'
+        #branch: master
+        tag: v3.9.0
+    buildsystem: simple
+    config-opts:
+      - --disable-mediactrl
+    build-commands:
+      - ./create_build_files4.sh
+      - sed -i "44 a \#include <cstdint>" src/md5/md5.hh
+      - make config=release -C build/3.0/gmake
+      - mv output/bin/wxformbuilder /app/bin/
+      - mkdir /app/share/wxformbuilder/
+      - mv output/xml/ /app/share/wxformbuilder/
+      - mv output/resources/ /app/share/wxformbuilder/
+      - mv output/plugins/ /app/share/wxformbuilder/
+      - mv output/*.* /app/share/wxformbuilder/
+      - mv output/lib/* /app/lib/
+      - mkdir /app/share/applications/
+      - mv install/linux/data/gnome/usr/share/applications/wxformbuilder.desktop /app/share/applications/org.wxformbuilder.wxFormBuilder.desktop
+      - sed -i 's|wxformbuilder.png|org.wxformbuilder.wxFormBuilder.png|' /app/share/applications/org.wxformbuilder.wxFormBuilder.desktop
+      - mkdir -p /app/share/icons/
+      - mv install/linux/data/gnome/usr/share/pixmaps/wxformbuilder.png /app/share/icons/org.wxformbuilder.wxFormBuilder.png
+      - mkdir /app/share/appdata/
+      - mv install/linux/data/gnome/usr/share/appdata/org.wxformbuilder.wxFormBuilder.appdata.xml /app/share/appdata/
+    cleanup:
+      - '*'
+
+  #-------------------------------------------------------------------------------
+  # yat
+  #-------------------------------------------------------------------------------
+  - name: yat
+    sources:
+      - type: dir
```

Comment:
so where are the sources? this won't work.

Reply:
Added sources the few lines below:
```
      - type: git
        url: 'https://gitlab.com/neva_blyad/yat'
        branch: master
```

Reply:
Fixed